### ISSUE TITLE
Capistrano deploy fails if no compile assets present

### ISSUE THREAD

**Issue by cbillen** [rollbar/rollbar-gem#738]
In our project we do not use the asset pipeline, the 2.15.6 version of the gem seems to require this, here's the exception trace we get if we try to deploy with 2.15.6, went back to 2.15.5 

```

Don't know how to build task 'deploy:compile_assets' (see --tasks)
Did you mean?  deploy:migrate
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task_manager.rb:59:in `[]'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task.rb:382:in `[]'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/capistrano-3.10.2/lib/capistrano/dsl/task_enhancements.rb:12:in `after'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rollbar-2.15.6/lib/rollbar/tasks/rollbar.cap:58:in `block in <top (required)>'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task_manager.rb:224:in `in_namespace'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/dsl_definition.rb:141:in `namespace'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rollbar-2.15.6/lib/rollbar/tasks/rollbar.cap:57:in `<top (required)>'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rollbar-2.15.6/lib/rollbar/capistrano3.rb:1:in `load'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rollbar-2.15.6/lib/rollbar/capistrano3.rb:1:in `<top (required)>'
/Users/cbillen/Projects/heatwave/Capfile:19:in `require'
/Users/cbillen/Projects/heatwave/Capfile:19:in `<top (required)>'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/rake_module.rb:29:in `load'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/rake_module.rb:29:in `load_rakefile'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:703:in `raw_load_rakefile'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:104:in `block in load_rakefile'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:186:in `standard_exception_handling'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:103:in `load_rakefile'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:82:in `block in run'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:186:in `standard_exception_handling'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:80:in `run'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/capistrano-3.10.2/lib/capistrano/application.rb:14:in `run'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/capistrano-3.10.2/bin/cap:3:in `<top (required)>'
/Users/cbillen/.rbenv/versions/2.5.0/bin/cap:23:in `load'
/Users/cbillen/.rbenv/versions/2.5.0/bin/cap:23:in `<top (required)>'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `load'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `kernel_load'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:28:in `run'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/cli.rb:424:in `exec'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/cli.rb:27:in `dispatch'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/cli.rb:18:in `start'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/exe/bundle:30:in `block in <top (required)>'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/lib/bundler/friendly_errors.rb:122:in `with_friendly_errors'
/Users/cbillen/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/bundler-1.16.1/exe/bundle:22:in `<top (required)>'
/Users/cbillen/Projects/heatwave/bin/bundle:3:in `load'
/Users/cbillen/Projects/heatwave/bin/bundle:3:in `<main>'
```

**Comment by cbillen** [comment]
The problem is that there’s an assumption about compile assets being present then doing the source map upload, but because we use webpack externally as part we leave this step to webpack to handle the sourcemap.  Maybe there should be a test to make sure deploy assets is present before trying this post deploy:compile_assets callback?

**Comment by ArturMoczulski** [comment]
This has been addressed in `v2.16.1` by removing source map uploads for now
